examples/how_to/get_started.py

```diff
@@ -251,18 +295,19 @@
 qm_params["drift"]["interval_s"] = 2
 qm_params["drift"]["min_spikes_per_interval"] = 2
 
-qm = sqm.compute_quality_metrics(we_TDC, qm_params=qm_params)
+qm = sqm.compute_quality_metrics(sa_TDC, qm_params=qm_params)
```

Comment:
Do we want to make quality metrics seem like this different thing or would it be better to do something like

sa_TDC.compute('quality_metrics').... and add in the additional arguments?

Reply:
Yes, much better for it to be analyzer.compute() notation! Done below